db/database.go

```diff
@@ -273,6 +288,23 @@ func (d *SqlxDatabase) getAuroraVersion() (string, error) {
 	return v.(string), err
 }
 
-func (d *SqlxDatabase) Stats() sql.DBStats {
-	return d.db.Stats()
+func (d *PgxDatabase) Stats() sql.DBStats {
+	stat := d.db.Stat()
+	return sql.DBStats{
+		MaxOpenConnections: int(stat.MaxConns()),
+		// Includes conns being constructed, so this violates the invariant documented
+		// in DBStats that OpenConnections = InUse + Idle.
+		OpenConnections: int(stat.TotalConns()),
+		InUse:           int(stat.AcquiredConns()),
+		Idle:            int(stat.IdleConns()),
+		// WaitCount is the number of connections for which sql had to wait.
+		// EmptyAcquireCount is the number of connections that pgx acquired after
+		// waiting.  The two are close enough (but race each other).
+		WaitCount: stat.EmptyAcquireCount(),
+		// Time to acquire is close enough to time spent waiting; fudge.
+		WaitDuration:      stat.AcquireDuration(),
+		MaxIdleClosed:     0, // TODO(ariels): Does pgx have this anywhere?
+		MaxIdleTimeClosed: 0, // TODO(ariels): Could generate this with post-connect/release hooks
+		MaxLifetimeClosed: 0, // TODO(ariels): Is this even possible in pgx?
```

Comment:
Check with Yoni

Reply:
Paging @johnnyaug for advice here?

Reply:
Cleared by @johnnyaug : not part of callhome, no known users of field on monitoring (or indeed cause to use it), at least one of these dbstats fields appears to have been added since written (but no easy way to tell - `database/sql` appears to have no version in `go.sum` because it's part of Go).